src/App.tsx

```diff
@@ -121,18 +111,18 @@ const App: React.FC = () => {
         {user.isLoggedIn && <AdminMenu />}
         <Breadcrumbs />
         {showAuthorityOrcidModal && <AuthorityOrcidModal />}
-        <StyledPageBody>
+        <ContentPage>
           <Switch>
             <Route exact path="/" component={Dashboard} />
-            {user.isLoggedIn && <Route exact path="/publications" component={Workspace} />}
-            {user.isLoggedIn && <Route exact path="/publications/new" component={NewPublication} />}
+            {user.isLoggedIn && <Route exact path="/new-publication" component={NewPublication} />}
+            {user.isLoggedIn && <Route exact path="/my-publications" component={MyPublications} />}
```

Comment:
valgte å gå for hypen i url.   https://stackoverflow.com/questions/10302179/hyphen-underscore-or-camelcase-as-word-delimiter-in-uris